src/Symfony/Component/VarExporter/ProxyHelper.php

```diff
@@ -200,10 +200,17 @@ public static function generateLazyProxy(?\ReflectionClass $class, array $interf
         return <<<EOPHP
             {$parent} implements \\{$interfaces}
             {
-                use \Symfony\Component\VarExporter\LazyProxyTrait;
+                use \Symfony\Component\VarExporter\LazyProxyTrait {
+                    __unserialize as __doUnserialize;
+                }
 
                 private const LAZY_OBJECT_PROPERTY_SCOPES = {$propertyScopes};
-            {$body}}
+            {$body}
+                public function __unserialize(\$data): void
+                { 
+                    \$this->__doUnserialize(\$data);
```

Comment:
I'd do all this only if the parent class doesn't have a type-hint!

Reply:
done